FoxHound.Client/src/.env

```diff
@@ -1 +1,2 @@
-API_URL=https://localhost:44360/api
\ No newline at end of file
+API_URL=https://localhost:44360/api
+APP_INSIGHTS_INSTRUMENTATION_KEY=
```

Comment:
Does leaving the instrumentation key null cause any issues when it's not found (i.e. running locally?

Reply:
No issues when running locally without it.  It simply will not push App Insights metrics anywhere.